Src/HostingStartup/HostingStartup/HostingStartup.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" />
```

Comment:
Do HostingStartup and Web need that fix? I remember they were importing those through common.

Reply:
They work without the extra import. Very confusing.